src/components/FiltersBar/FiltersBar.styles.tsx

```diff
@@ -16,21 +19,52 @@ export const MobileFilterContainer = styled.div`
 `
 
 export const FiltersContainer = styled.div<{ open: boolean }>`
+  align-items: center;
+  margin: 0 var(--size-global-horizontal-padding);
+  background-color: ${colors.gray[700]};
   display: ${({ open }) => (open ? 'flex' : 'none')};
   justify-content: space-between;
-  padding: ${sizes(4)};
-  background-color: ${colors.gray[900]};
-  will-change: opacity;
-  transition: opacity ${transitions.timings.sharp} ${transitions.easing};
+  padding: ${sizes(4)} var(--size-global-horizontal-padding);
+  will-change: opacity, height;
```

Comment:
the open and close anim is looking a bit rough because of the padding doesn't allow to fully have 0 height this happened to me as well when I tried to implement so I'm not sure what to do.

I would use react-spring but Im sure @kdembler wouldnt like that

Reply:
I agree that the current animation doesn't look great but I don't think `react-spring` is the way to go here. I think the issue is that we're animating `height` and `opacity` but we could just `transform` the element instead. It could hide underneath the tabs bar and slide from there, right? This should fix issues with `height` transition

Reply:
But let's do any work on this in a separate PR, to restrict the scope here

Reply:
alternatively, a div wrapper could be added and the animation could be done there as is, I think as long as the wrapper doesn't have padding it should work tho I haven't tried it, the advantage being not having to touch z-index

Reply:
That's also a possible solution but animating `height` will always be more rough than animating `transform`

Reply:
@kdembler @eldiegod I did a quick experiment with animating the `height` and it turns out that if we make the animation quick enough, we might actually get away with that rough and sudden change in the height not being animated. Here's the result: https://jsfiddle.net/toiletgranny/1snkh2bL/93/

If you decide on going with the above, then I'd suggest using the `transition.fast` animation token.

Another, "bonus" thing I discovered today is that if we'll use `position: sticky`, then the change in the overall height of the filters bar doesn't seem to affect the user's scroll position. That's pretty cool! <3

Reply:
@toiletgranny I've did transform transition, take a look. Unfortunately `position: sticky` didn't work well in this case.

Reply:
@wyrdrender Oh man, it looks so pretty right now! It's not 100% what we had in that Figma prototype but I don't mind at all, I might even like it more than the original! 

In case you're still on it, then I'd suggest dropping the opacity animation (it's not needed right now).